lemur/certificates/service.py

```diff
@@ -939,3 +940,68 @@ def get_issued_cert_count_for_authority(authority):
     :return:
     """
     return database.db.session.query(Certificate).filter(Certificate.authority_id == authority.id).count()
+
+
+def get_all_valid_certificates_with_source(source_id):
+    """
+    Return list of certificates
+    :param source_id:
+    :return:
+    """
+    return (
+        Certificate.query.filter(Certificate.sources.any(id=source_id))
+        .filter(Certificate.revoked == false())
+        .filter(Certificate.not_after >= arrow.now())
+        .filter(not_(Certificate.replaced.any()))
+        .all()
+    )
+
+
+def get_all_valid_certificates_with_destination(destination_id):
+    """
+    Return list of certificates
+    :param destination_id:
+    :return:
+    """
+    return (
+        Certificate.query.filter(Certificate.destinations.any(id=destination_id))
+        .filter(Certificate.revoked == false())
+        .filter(Certificate.not_after >= arrow.now())
+        .filter(not_(Certificate.replaced.any()))
+        .all()
+    )
+
+
+def remove_source_association(certificate, source):
+    certificate.sources.remove(source)
+    database.update(certificate)
+
+    metrics.send(
+        "delete_certificate_source_association",
+        "counter",
+        1,
+        metric_tags={"status": SUCCESS_METRIC_STATUS,
+                     "source": source.label,
+                     "certificate": certificate.name}
+    )
+
+
+def remove_destination_association(certificate, destination):
+    certificate.destinations.remove(destination)
+    database.update(certificate)
+
+    try:
+        remove_from_destination(certificate, destination)
+    except Exception as e:
+        # This cleanup is the best-effort, it will capture the exception and log
+        sentry.captureException()
```

Comment:
I just realized we have completed the Sentry migration. Would you mind make the following changes:
```
- from lemur.extensions import sentry
+ from sentry_sdk import capture_exception
- sentry.captureException()
+ capture_exception()
```